src/org/traccar/processing/CopyAttributesHandler.java

```diff
@@ -37,7 +37,12 @@ protected Position handlePosition(Position position) {
         if (attributesString.isEmpty()) {
             attributesString = Position.KEY_DRIVER_UNIQUE_ID;
         } else {
-            attributesString += "," + Position.KEY_DRIVER_UNIQUE_ID;
+            if (attributesString.equals("all")) {
```

Comment:
This is not related to computed attributes in any way. Also, setting attributes would clear existing attributes, so I don't think it's correct. Also, I don't think using special keyword is a good idea. We need some different way of copying all attributes. For example, if `copyAttributes` is not specified it would mean copy app.

Reply:
No its not related. Just happened I made the changes on the same branch by accident.

Line 36, 'if (attributesString.isEmpty()) {'
My understanding of 'lookupAttributeString' is that the default is "" empty. and would match line 36. Is that incorrect?

Reply:
I'm proposing to use absence of the parameter, not empty value.

Reply:
More like this, if null then do as previous isEmpty, if specified but empty then its all else its limited?
```
        String attributesString = Context.getDeviceManager().lookupAttributeString(
                position.getDeviceId(), "processing.copyAttributes", null, true);
        Position last = getLastPosition(position.getDeviceId());
        if (attributesString == null) {
            attributesString = Position.KEY_DRIVER_UNIQUE_ID;
        } else {
            if (attributesString.isEmpty()) {
                position.setAttributes(last.getAttributes());
                return position;
            } else {
                attributesString += "," + Position.KEY_DRIVER_UNIQUE_ID;
            }
        }
```

Reply:
Something like it.